eclipse/sw360
# feat(ui-rest): Provided option in attachment usage to include/exclude concluded licenses during LicenseInfo Generation

> Provided option in attachment usage to include/exclude concluded licenses during LicenseInfo Generation
> * Which issue is this pull request belonging to and how is it solving it? (*#1074*)
> * Did you add or update any new dependencies that are required for your change?

### How To Test? - Refer #1074 
> - Change Attachment Usage with different file types from Attachment Tab and From LicenseInfo/Report Generation Page.
      - Verify Attachement Usage with proper info are getting created and deleted from DB based on selection. 
> - Old Attachement Usages should function normally - by default conclusions are included.
> - Download LicenseInfo and Report in different format with/without conclusions from UI and Rest

> Have you implemented any additional tests? - No

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
